AttitudeManager/Inc/PID.hpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * PID.hpp
+ *
+ *  Created on: May 27, 2022
+ *      Author: Anthony (anni) Luo
+ */
+
+#ifndef PID_HPP_
+#define PID_HPP_
```

Comment:
nit: `PID_HPP` I think based off the existing files. Or not, but we should be consistent

Reply:
autogenerated header guards go brrt oops